src/api.mock.ts

```diff
@@ -3,13 +3,114 @@
+    rating: 5,
+    platforms: ["pc"],
+    minAge: 8,
+    addDate: new Date("2023-01-01"),
+    description:
+      "Minecraft is a game made up of blocks, creatures, and community. Blocks can be used to reshape the world or build fantastical creations. Creatures can be battled or befriended, depending on your playstyle. Experience epic adventures solo or with friends, there’s no wrong way to play.Unless you’re digging straight down.",
+  },
+  {
+    id: 5,
+    name: "Minecraft",
+    price: 25.99,
+    image: "https://raw.githubusercontent.com/ddushev/Vention/feat/home-page/src/publicMock/minecraft.jpg",
+    rating: 5,
+    platforms: ["pc", "ps5", "xbox"],
+    minAge: 3,
+    addDate: new Date("2023-01-05"),
+    description:
+      "Minecraft is a game made up of blocks, creatures, and community. Blocks can be used to reshape the world or build fantastical creations. Creatures can be battled or befriended, depending on your playstyle. Experience epic adventures solo or with friends, there’s no wrong way to play.Unless you’re digging straight down.",
+  },
+  {
+    id: 6,
+    name: "Overwatch",
+    price: 23.99,
+    image: "https://raw.githubusercontent.com/ddushev/Vention/feat/home-page/src/publicMock/overwatch.jpg",
+    rating: 4,
+    platforms: ["pc"],
+    minAge: 5,
+    addDate: new Date("2023-01-06"),
+    description:
+      "Overwatch 2 is a free-to-play shooter featuring 30+ epic heroes, each with game-changing abilities. Choose your hero, group up with your friends and battle across all-new maps and modes in the ultimate team-based shooter.",
+  },
+  {
+    id: 7,
+    name: "Sims 4",
+    price: 30.99,
+    image: "https://raw.githubusercontent.com/ddushev/Vention/feat/home-page/src/publicMock/sims4.jpg",
+    rating: 5,
+    platforms: ["pc"],
+    minAge: 2,
+    addDate: new Date("2023-01-01"),
+    description:
+      "Minecraft is a game made up of blocks, creatures, and community. Blocks can be used to reshape the world or build fantastical creations. Creatures can be battled or befriended, depending on your playstyle. Experience epic adventures solo or with friends, there’s no wrong way to play.Unless you’re digging straight down.",
+  },
+  {
+    id: 8,
+    name: "Terraria",
+    price: 4.99,
+    image: "https://raw.githubusercontent.com/ddushev/Vention/feat/home-page/src/publicMock/terraria.jpg",
+    rating: 3,
+    platforms: ["pc", "ps5", "xbox"],
+    minAge: 4,
+    addDate: new Date("2023-01-04"),
+    description:
+      "The very world is at your fingertips as you fight for survival, fortune, and glory. Delve deep into cavernous expanses, seek out ever-greater foes to test your mettle in combat, or construct your own city - In the World of Terraria, the choice is yours!",
+  },
+];
 
 export default webpackMockServer.add((app) => {
-  app.get(apiEndpoints.testMock, (_req, res) => res.json(mockData));
+  app.get(`${apiEndpoints.searchMock}`, (_req, res) => {
```

Comment:
Such syntax doesn't make sense. It makes redundant efforts for JS-engine
Actual `app.get(`${apiEndpoints.searchMock}`, (_req, res) => {`
Expected `app.get(apiEndpoints.searchMock, (_req, res) => {`